samples_project/Assets/SampleViewer/Samples/Routing/readme.md

```diff
@@ -1,15 +1,23 @@
-# Routing
+# Find a route
 
 Show a route between two points on a map using Esri's routing service REST API.
 
 ![Routing](routing.png)
 
+## How to use the sample
+
+This sample uses Esri's [routing service's REST API](https://developers.arcgis.com/rest/network/api-reference/overview-of-network-analysis-services.htm) to query the closest route along the road network between two points. This service uses routing operations associated with your API Key. You can learn more about [API keys](https://developers.arcgis.com/documentation/mapping-apis-and-services/security/api-keys/) and [Accounts](https://developers.arcgis.com/documentation/mapping-apis-and-services/deployment/accounts/) in the _Mapping APIs and location services_ guide.
+
+1. Click on the **RouteManager** game object under **ArcGISMap** in the **Hierarchy** window.
+2. Set your API key in the **Inspector** window.
+3. While holding shift, left click on the map twice. (This sample is only set up to work with mouse and keyboard.)
```

Comment:
```suggestion
1. Open the **Routing** level.
2. Click on the **RouteManager** game object under **ArcGISMap** in the **Hierarchy** window.
3. Set your API key in the **Inspector** window.
4. While holding shift, left click on the map twice. (This sample is only set up to work with mouse and keyboard.)
```

Reply:
@Jade-JadeH @avr757 I'm not thinking of changing the sample level name but the sample name on doc.
So I think it's better to include the name of the sample level. What do you think?